keps/sig-node/3857-rro-mounts/README.md

```diff
@@ -711,7 +732,8 @@ in back-to-back releases.
 
 #### GA
 - Two beta releases of Kubernetes at least
-- containerd, CRI-O, and cri-dockerd supports the feature with their GA releases
+- containerd (v2.0) and CRI-O (v1.30) support the feature with their GA releases.
+  The feature has been implemented in the `master` branch of cri-dockerd too.
```

Comment:
Discussed in:
- https://github.com/kubernetes/enhancements/issues/3857#issuecomment-2637368278